app/Resources/assets/js/lsdoc/index.js

```diff
@@ -556,6 +556,134 @@ function listRepositories(){
     $('#back').html('');
 }
 
+var CommentSystem = (function(){
```

Comment:
How can I move this section in a different JS file and load it when the commenting is turned on? do I have to make some change in the gulpfile?